.github/workflows/publish_docs_to_wiki.yml

```diff
@@ -0,0 +1,36 @@
+name: Publish docs to Wiki
+
+on:
+  push:
+    paths:
+      - docs/**
+    branches:
+      - main
+      
+env:
+  USER_TOKEN: ${{ secrets.WIKI_ACTION_TOKEN }}
+  USER_NAME: team-nimblehq
+  USER_EMAIL: dev@nimblehq.co
+  OWNER: ${{ github.event.repository.owner.name }}
+  REPOSITORY_NAME: ${{ github.event.repository.name }}
+
+jobs:
+  Publish-Docs-To-Wiki:
+    runs-on: ubuntu-latest
+    steps:  
+    - uses: actions/checkout@v2
+    - name: Prepare
+      run: |
+         mkdir tmp_wiki
+         cd tmp_wiki
+         git init
+         git config user.name $USER_NAME
+         git config user.email $USER_EMAIL
+         git pull https://$USER_TOKEN@github.com/$OWNER/$REPOSITORY_NAME.wiki.git
+    - name: Push
+      run: |
+        rsync -av --delete docs/ tmp_wiki/ --exclude .git
+        cd tmp_wiki
+        git add .
+        git commit -m "Update Wiki content"
```

Comment:
I am curious that all the commits to update wiki will be `"Update Wiki content"` 🤔  So from the wiki history page, the history will show the same for all the updates? 🤔

Reply:
it's an auto bump message 🤔  so I think it's ok

Reply:
Another possibility would be the get the first commit message from `git log` 🙏

Reply:
Reference: [link](https://github.com/Andrew-Chen-Wang/github-wiki-action/blob/master/entrypoint.sh)

![Uploading Screen Shot 2564-08-31 at 10.27.37.png…]()

Reply:
but from a normal development flow, 1st message from gitlog will likely be the merge message or a commit message from an unrelated task to this WIKI update no? 🤔

Reply:
Ouch, true that! It's gonna get quite complicated to find a the first commit message that relates to Wiki then 🙈